app/(dashboard)/(routes)/favorites/components/favorites-list.tsx

```
@@ -0,0 +1,33 @@
+
+  if (favoriteIds.length === 0) {
+    return (
+      <div className="text-center mt-10">
+        <p className="text-muted-foreground">No favorite hackathons yet.</p>
+      </div>
+    );
+  }
+
+  return (
+    <div className="text-center mt-10">
+      <p className="text-muted-foreground">External favorite hackathons are displayed in the Hackathons page under the "Favorites" tab.</p>
```

ESLint

[error] `"` can be escaped with `&quot;`, `&ldquo;`, `&#34;`, `&rdquo;`.

(react/no-unescaped-entities)